runners/metrics_client_test.go

```diff
@@ -30,3 +32,29 @@ func (c *prometheusClientMock) setResponce(key types.NamespacedName, stats *Volu
 	}
 	c.stats[key] = stats
 }
+
+var _ = Describe("test prometheusClient", func() {
+	It("test metrics", func() {
+		c, err := NewPrometheusClient("http://noconnection")
```

Comment:
I think this is not good. Would you make a fake server that returns always 404.
Ex. use `httptest.NewServer` and `http.NotFound`

Reply:
fixed it.